Directory.Build.targets

```diff
@@ -1,8 +1,8 @@
 <Project>
   <Import Project="src\_PublishArtifacts.Build.props" Condition="$(IsPackable) == True"/>
-  <PropertyGroup Condition="$(IsPackable) == True OR $(MSBuildProjectName.StartsWith('Tests'))">
+  <PropertyGroup Condition="$(SignAssembly) == True OR $(IsPackable) == True OR $(MSBuildProjectName.StartsWith('Tests'))">
     <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>$(SolutionDir)\build\keys\keypair.snk</AssemblyOriginatorKeyFile>
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)\build\keys\keypair.snk</AssemblyOriginatorKeyFile>
```

Comment:
$(SolutionDir) is only defined inside an IDE, $(MSBuildThisFileDirectory) is defined by msbuild.

Relative path would have worked as well but may look ambiguous.